src/electron/package_linux.action.sh

```diff
@@ -13,11 +13,32 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+PLATFORM=
```

Comment:
Delete platform

Reply:
ah, i saw this pattern elsewhere and assumed there was some scoping going on. good to know YAGNI